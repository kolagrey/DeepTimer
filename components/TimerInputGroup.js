import React from 'react';
import { View } from 'react-native';
import styles from '../styles';

const TimerInputGroup = (props) => {
    return (
        <View style={styles.inputContainer}>
            {props.children}
        </View>
    );
};

export default TimerInputGroup;