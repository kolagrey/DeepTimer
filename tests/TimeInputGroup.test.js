import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';

import TimerInputGroup from '../components/TimerInputGroup';

describe('<TimerInputGroup/>', () => {

    test('renders correctly', () => {
        const tree = renderer.create(<TimerInputGroup/>).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it("is of type 'View'", () => {
        const tree = renderer
            .create(
                <TimerInputGroup />,
            ).toJSON();
        expect(tree.type).toBe('View');
    });

    it('can hold a child', () => {
        const tree = renderer
            .create(
                <TimerInputGroup>
                    <Text>
                        Hello Test
                    </Text>
                </TimerInputGroup>,
            ).toJSON();
        expect(tree.children.length).toBe(1)
    });
});