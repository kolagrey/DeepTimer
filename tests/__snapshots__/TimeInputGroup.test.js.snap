// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimerInputGroup/> renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "flex": 1,
      "flexDirection": "row",
      "padding": 10,
    }
  }
/>
`;
